frontend/test/metabase/scenarios/models/helpers/e2e-models-helpers.js

```diff
@@ -61,7 +61,7 @@ export function selectDimensionOptionFromSidebar(name) {
 }
 
 export function openDetailsSidebar() {
-  cy.findByTestId("saved-question-header-button").click();
+  cy.findByTestId("saved-question-header-title").click();
```

Comment:
The test ID is changed in this commit. https://github.com/metabase/metabase/commit/ee25189dd8073dc7e4a51b0a0ab15fcade3bf7ce#diff-f66769a8ce4d54035a928c1eb05e282e1277a7e6c582139b0218f73160066506R36